pkg/awsclient/mock/zz_generated.mock_client.go

```diff
@@ -1,5 +1,5 @@
 // Code generated by MockGen. DO NOT EDIT.
-// Source: ./client.go
+// Source: pkg/awsclient/client.go
```

Comment:
@ArielLima this looks like you got hit with the same-ish `generate` issue that I was running into a few weeks ago.

Try using `./boilerplate/_lib/container-make generate` and then commit those changes.